specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Quantum Client",
+    "description": "Azure Quantum REST API client",
+    "version": "2019-11-04-preview",
+    "x-ms-code-generation-settings": {
+      "name": "QuantumClient"
+    }
+  },
+  "host": "app-jobscheduler-prod.azurewebsites.net",
```

Comment:
Is that the prod host for everybody that uses your service?

Reply:
For now it is. We're still working on single end-point and multi region support.